### ISSUE TITLE
segmentation fault

### ISSUE THREAD

**Issue by yangze01** [codertimo/BERT-pytorch#16]
When the code's Dataset class is modified, the error occurs. I try to debug the error, the code except at line: loss.backward().
![image](https://user-images.githubusercontent.com/11796873/47277153-bd9ec880-d5ef-11e8-8e40-6d80a5e55749.png)

**Comment by yangze01** [comment]
@codertimo thank you very much. you are right, it's CUDA's problem. your code with my own cooked data is running now. it's so cool to reproduce the BERT so quickly, thanks for your sharing!

**Comment by yangze01** [comment]
@codertimo thank you very much. You are right, it's the CUDA's problem. the code is running with my own cooked data. It's so cool to reproduce BERT so quickly, thanks for your sharing!
